synfig-core/src/modules/mod_png/trgt_png_spritesheet.cpp

```diff
@@ -479,6 +478,7 @@ png_trgt_spritesheet::write_png_file()
     png_write_info_before_PLTE(png_ptr, info_ptr);
     png_write_info(png_ptr, info_ptr);
 
+	unsigned char* buffer = new unsigned char[4 * sheet_width];
```

Comment:
Maybe unique_ptr?
It would avoid problems with `delete[] buffer`.

Reply:
Done.